resources/js/alternatingCase.js

```diff
@@ -0,0 +1,26 @@
+var toUpper = require('./toUpper'); // requiring custom toUpper method
+var toLower = require('./toLower'); // requiring custom toLower method
+
+// setting alternatingCase method to string protoype
+
+String.prototype.alternatingCase = function() {
+  var checkString = /\w/g; // setting regex pattern to match agains't string
+
+  /**
```

Comment:
I see what you are trying to do here with all the comments. it's nice and great but not really scalable. The idea is to properly describe what the code is doing. Keep comments descriptive; don't write an essay out of them.